Ron Barrantes

ronbarrantes opened the pull request:
Submitting a branch other than master

ronbarrantes committed:
start of the project

ronbarrantes committed:
account model done

ronbarrantes committed:
Working on my tests

ronbarrantes committed:
test changes

ronbarrantes committed:
working on the aws image uploader

ronbarrantes committed:
Adding the test files

ronbarrantes committed:
working on my tests still

ronbarrantes committed:
fix misspeled errors

ronbarrantes committed:
Modified files

ronbarrantes committed:
commented out expects on image-router

vladimirsan commented on lab-ron/__test__/auth-router.test.js:
```diff
@@ -0,0 +1,78 @@
+
+  describe('POST /signup', () => {
+
+    test('POST /signup returning 200 OK', () => {
```
Add a little bit more detail about when each test should succeed.

vladimirsan replied:
Also, missing 409 test.

vladimirsan commented on lab-ron/route/profile-router.js:
```diff
@@ -0,0 +1,28 @@
+const Profile = require('../model/profile.js');
+
+module.exports = new Router()
+  .post('/profiles', bearerAuth, jsonParser, (req, res, next) => {
```
No GET route

vladimirsan commented on lab-ron/__test__/profile-router.test.js:
```diff
@@ -0,0 +1,60 @@
+
+  describe('POST /profiles', () => {
+
+    test('200 should return a profile', () => {
```
No 400 test

vladimirsan commented on lab-ron/route/image-router.js:
```diff
@@ -0,0 +1,49 @@
+const upload = multer({ dest: `${__dirname}/../temp` });
+
+module.exports = new Router()
+  .post('/images', bearerAuth, upload.any(), (req, res, next) => {
```
No DELETE method

ronbarrantes committed:
finished the auth router

ronbarrantes committed:
get profile with fuzzies

ronbarrantes committed:
Profile Routes done

ronbarrantes committed:
added a bunch of stuff

got my profile router working
fix the setup mock for aws-s3-mock
working on image router

ronbarrantes committed:
working on the README

ronbarrantes committed:
All done

ronbarrantes committed:
tweak one of the messages for bearere auth
